packages/@aws-cdk/cloudformation-include/test/test-templates/fn-sub-valid-escaping.json

```diff
@@ -0,0 +1,10 @@
+{
+  "Resources": {
+    "Bucket": {
+      "Type": "AWS::S3::Bucket",
+      "Properties": {
+        "BucketName": { "Fn::Sub": "some-bucket${!AWS::AccountId}7896${ ! AWS::Region}${!Immediate}234" }
```

Comment:
Again, I have no idea what this test is supposed to verify.

Is it that `${ ! AWS::Region}` is correctly escaped? Then use a reference that is invalid there! Like `${  ! DoesNotExist}`.

Also, don't use the word `valid` in the name of the file - all templates are valid in this directory, so it's redundant.